v1/accounts/all.md

```diff
@@ -1,10 +1,14 @@
+
 # Accounts API v1.0.0
 
 Provides CRUD operations on the accounts resource.
 
-* Host ``
+*
+Host ``
+EOL
```

Comment:
this doesn't look right, I remember fixing this bug already. Could you regenerate everything by running ./generate.sh script?

Reply:
Might need to run ./deps.sh to update binaries?